### ISSUE TITLE
v4.0.5 contains pure ES2015 modules

### ISSUE THREAD

**Issue by zxqfox** [chaijs/type-detect#127]
```
$ grep version packages/postcss-tone-vars/node_modules/type-detect/package.json
  "version": "4.0.5"
$ grep export packages/postcss-tone-vars/node_modules/type-detect/index.js
export default function typeDetect(obj) {
```

Feels like it requires rebuild ;-(

**Comment by zxqfox** [comment]
Looks like it was already fixed in 7900eaf5c8568edf2557810355544f49a064dd65

My bad, I'll reinstall all the things. Thank you!

**Comment by vieiralucas** [comment]
Btw yes `index.js` contains `export default...` but if you run the `main` of `package.json` is not `index.js`
Look at print:
![image](https://user-images.githubusercontent.com/7764293/32630726-e2802ee0-c584-11e7-81f9-6102161af11b.png)
